acceptance/analyzer_test.go

```diff
@@ -960,25 +987,24 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 						// Don't attempt to remove the image, as it's stored in the test registry, which is ephemeral.
 						// Attempting to remove the image sometimes produces `No such image` flakes.
 
-						it("restores cache metadata", func() {
+						it("throw read/write error", func() {
```

Comment:
I think you could change the name of the context from `when("no auth registry", func() {` to `when("read only registry", func() {`

Reply:
```suggestion
						it("throws read/write error", func() {
```